core/documents-manager.php

```diff
@@ -719,4 +702,42 @@ public static function get_create_new_post_url( $post_type = 'page', $template_t
 
 		return $new_post_url;
 	}
+
+	private function get_doc_type_by_id( $post_id ) {
+		// Content built with Elementor.
+		$template_type = $this->get_template_type_by_id( $post_id );
+
+		if ( $template_type && isset( $this->types[ $template_type ] ) ) {
+			return $template_type;
+		}
+
+		// Auto-save post type should be inherited from the original post.
+		if ( wp_is_post_autosave( $post_id ) ) {
+			$post_id = wp_get_post_parent_id( $post_id );
+		}
```

Comment:
Maybe move to the top and delete the related code on `get_template_type_by_id`

Reply:
Done